### PR TITLE
Minor refactoring of JettyResponseFuture

### PR DESCRIPTION
**Pull request by nezihyigitbasi**

### CODE REVIEW

**Comment by electrum** [comment]
Is this just cleanup? Doesn’t appear to have any functional changes

**Comment by nezihyigitbasi** [comment]
Yep, just cleanup. I updated the commit title to make it clear.

File: http-client/src/main/java/io/airlift/http/client/jetty/JettyResponseFuture.java [hunk 1]
```diff
@@ -85,7 +81,7 @@ protected void completed(Response response, InputStream content)
             return;
         }
         state.set(JettyAsyncHttpState.DONE);
-        set(value);
+        super.set(value);
```
Comment (electrum) [inline http-client/src/main/java/io/airlift/http/client/jetty/JettyResponseFuture.java]: This qualifier is redundant

**Comment by electrum** [comment]
Merged, thanks!

**Pull request closed without merging**